### PR TITLE
update API url and ledger account

### PR DESCRIPTION
**Pull request by nvxtien**

### CODE REVIEW

File: src/main/java/io/tatum/ledger/Account.java [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+import java.util.concurrent.ExecutionException;
+
+import static io.tatum.constants.Constant.TATUM_API_URL;
+
+public class Account {
+
+    /**
+     * For more details, see <a href="https://tatum.io/apidoc#operation/getAccountByAccountId" target="_blank">Tatum API documentation</a>
+     */
+    public io.tatum.model.response.ledger.Account getAccountById(String id) throws IOException, ExecutionException, InterruptedException {
```
Comment (tatumio) [inline src/main/java/io/tatum/ledger/Account.java:23]: do top level import please

File: src/main/java/io/tatum/utils/Async.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+import java.net.http.HttpResponse;
+import java.time.Duration;
+import java.util.concurrent.ExecutionException;
+
+public class Async {
+
+    public static String post(String uri, String apiKey, String requestBody) throws ExecutionException, InterruptedException {
+        var request = HttpRequest.newBuilder()
+                .uri(URI.create(uri))
+                .timeout(Duration.ofMinutes(1))
```
Comment (tatumio) [inline src/main/java/io/tatum/utils/Async.java:15]: Let's do 20 seconds timeout

**Review by tatumio — CHANGES_REQUESTED** [review]

File: src/main/java/io/tatum/ledger/Account.java (hunk 1 above)
Comment (nvxtien) [reply src/main/java/io/tatum/ledger/Account.java:23]: There are two classes with same name. So there is one importing explicitly. I think of changing the class name.

File: src/main/java/io/tatum/utils/Async.java (hunk 2 above)
Comment (nvxtien) [reply src/main/java/io/tatum/utils/Async.java:15]: Ok.

File: src/main/java/io/tatum/ledger/Account.java (hunk 1 above)
Comment (tatumio) [reply src/main/java/io/tatum/ledger/Account.java:23]: Let's change the name of the ledger/Account.java, not the model

File: src/main/java/io/tatum/ledger/Account.java [hunk 3]
```diff
@@ -20,18 +21,18 @@
     /**
      * For more details, see <a href="https://tatum.io/apidoc#operation/getAccountByAccountId" target="_blank">Tatum API documentation</a>
      */
-    public io.tatum.model.response.ledger.Account getAccountById(String id) throws IOException, ExecutionException, InterruptedException {
+    public BookkeepingAccount getAccountById(String id) throws IOException, ExecutionException, InterruptedException {
```
Comment (tatumio) [inline src/main/java/io/tatum/ledger/Account.java:24]: I would go with LedgerAccount instead

**Review by tatumio — CHANGES_REQUESTED** [review]

File: src/main/java/io/tatum/ledger/Account.java (hunk 3 above)
Comment (nvxtien) [reply src/main/java/io/tatum/ledger/Account.java]: Ok.

**Pull request merged**